src/i18n/hi/nav.ts

```diff
@@ -1,11 +1,74 @@
 import { NavDictionary } from '../translation-checkers';
 
 export default NavDictionary({
-	// Start Here
 	startHere: 'यहाँ से शुरू करे',
 	'getting-started': 'पहले कदम',
+	install: 'स्थापित करें',
+	'editor-setup': 'एडिटर सेटअप',
+	'guides/upgrade-to/v3': 'v3 में अपग्रेड करें',
 
-	// Core Concepts
-	coreConcepts: 'मूल अवधारणाएं',
+	coreConcepts: 'मूल अवधारणाएँ',
 	'concepts/why-astro': 'क्यों Astro',
+	'concepts/islands': 'Astro द्वीप',
+
+	tutorials: 'शिक्षण सत्र',
+	'blog-tutorial': 'ब्लॉग बनाएं',
+	'add-collections-tutorial': 'सामग्री संग्रह बढ़ाएं',
+	'add-transitions-tutorial': 'दृश्य परिसर बढ़ाएं',
+
+	basics: 'मौलिक अवधारणाएँ',
+	'core-concepts/project-structure': 'परियोजना संरचना',
+	'core-concepts/astro-components': 'घटक',
+	'core-concepts/astro-pages': 'पृष्ठ',
+	'core-concepts/layouts': 'नक़्शे',
+	'core-concepts/astro-syntax': 'Astro की सिंटैक्स',
+
+	builtins: 'सामग्री स्थापित',
+	'guides/content-collections': 'सामग्री संग्रह',
+	'guides/view-transitions': 'view transitions',
+
+	addons: 'एड-ऑन्स',
+	'guides/integrations-guide': 'एकीकरण जानकारी',
+	'core-concepts/framework-components': 'UI फ़्रेमवर्क्स',
+	'guides/server-side-rendering': 'सर्वर-साइड रेंडरिंग (SSR)',
+
+	examples: 'उदाहरण',
+	'guides/migrate-to-astro': 'Astro में माइग्रेट करें',
+	'guides/cms': 'एक CMS से कनेक्ट करें',
+	'guides/backend': 'बैकएंड सेवाओं को जोड़ें',
+	'guides/deploy': 'अपनी साइट को डिप्लॉय करें',
+	'guides/recipes': 'अधिक रेसिपी',
+
+	features: 'गाइड्स',
+	'core-concepts/routing': 'रूटिंग',
+	'guides/markdown-content': 'मार्कडाउन और MDX',
+	'guides/client-side-scripts': 'स्क्रिप्ट और घटना संचालन',
+	'guides/styling': 'स्टाइल और CSS',
+	'guides/images': 'छवियाँ',
+	'guides/fonts': 'फ़ॉन्ट्स',
+	'guides/imports': 'आयात',
+	'core-concepts/endpoints': 'एंडपॉइंट्स',
+	'guides/data-fetching': 'डेटा फ़ेचिंग',
+	'guides/internationalization': 'अंतर्राष्ट्रीयकरण',
+	'guides/middleware': 'मिडलवेयर',
+	'guides/testing': 'टेस्टिंग',
+	'guides/troubleshooting': 'समस्या समाधान',
```

Comment:
Technologies should be kept as it is (ex: React, Astro) as a general rule
```suggestion
	features: 'गाइड्स',
	'core-concepts/routing': 'रूटिंग',
	'guides/markdown-content': 'Markdown और MDX',
	'guides/client-side-scripts': 'स्क्रिप्ट और घटना संचालन',
	'guides/styling': 'स्टाइल और CSS',
	'guides/images': 'छवियाँ',
	'guides/fonts': 'फ़ॉन्ट्स',
	'guides/imports': 'आयात',
	'core-concepts/endpoints': 'एंडपॉइंट्स',
	'guides/data-fetching': 'डेटा फ़ेचिंग',
	'guides/internationalization': 'अंतर्राष्ट्रीयकरण',
	'guides/middleware': 'मिडलवेयर',
	'guides/testing': 'परिक्षण',
	'guides/troubleshooting': 'समस्या समाधान',
```